helmfile/charts/gatekeeper/constraints/values.yaml

```
@@ -10,8 +10,19 @@ requireResourceRequests:
 disallowedTags:
     enable: true
     enforcementAction: dryrun
+allowUserCRDs:
```

As a general comment I think it would make more sense to rename this everywhere to `restrictUserCRDs` since the opa policy is restricting things by default, not making things more open.
```suggestion
restrictUserCRDs:
```

---

Sure, `restrictUserCRDs` maybe makes more senses in a OPA view. The default behavior is already restricing users to use no CRDs. And enabling this allow them to use pre-defined crds. But i can change the name.